### PR TITLE
Feature/liquidity view

### PR DESCRIPTION
**Pull request by MrZyr0**

### CODE REVIEW

**Review by ChxGuillaume — APPROVED** [review]

File: src/components/RemoveLiquidity.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  try {
+    state.isRemovingLiquidity = true;
+    emit('transationStarted');
+
+    const lpTransation = await props.lpContract.removeLiquidity(toWei(state.lpTokenToRemove));
+    await lpTransation.wait();
+
+    emit('transationEnded');
+    state.isRemovingLiquidity = false;
+  } catch (error) {
```
Comment (MaxenceMottard) [inline src/components/RemoveLiquidity.vue:78]:
```suggestion
  } catch {
```

Comment (MaxenceMottard) [inline src/components/RemoveLiquidity.vue:78]:
```suggestion
  } catch {
```

File: src/router/index.js [hunk 2]
```diff
@@ -50,6 +58,14 @@ const router = createRouter({
...
       component: () => import('../views/Analytics/TokenView.vue'),
     },
+	{
+	  path: '/liquidity',
+      name: 'Liquidity',
+      // route level code-splitting
+      // this generates a separate chunk (About.[hash].js) for this route
+      // which is lazy-loaded when the route is visited.
+      component: () => import('../views/Liquidity/LiquidityView.vue'),
+	}
```
Comment (MaxenceMottard) [inline src/router/index.js:68]: remove

File: src/components/AddLiquidity.vue [hunk 3]
```diff
@@ -0,0 +1,160 @@
+<template>
```
Comment (MaxenceMottard) [inline src/components/AddLiquidity.vue:1]: move into Liquidity directory

File: src/components/RemoveLiquidity.vue [hunk 4]
```diff
@@ -0,0 +1,83 @@
+<template>
```
Comment (MaxenceMottard) [inline src/components/RemoveLiquidity.vue:1]: move into Liquidity directory

**Review by MaxenceMottard — APPROVED** [review]

**Review by MaxenceMottard — APPROVED** [review]

**Pull request merged**